.github/workflows/build-test.yml

```diff
@@ -241,11 +274,27 @@ jobs:
           crate: cargo-make
           version: latest
 
+      - name: Get Diplomat version
+        id: diplomat-version
+        run: |
+          echo "::set-output name=rev::$(cargo make diplomat-get-rev --loglevel error |  tr -d '[:space:]')"
+        shell: bash
+      - name: Attempt to load cached Diplomat
+        uses: actions/cache@v2
+        id: diplomat-cache
+        with:
+          path: |
+            ~/.cargo/bin/diplomat-tool
+            ~/.cargo/bin/diplomat-tool.exe
+          key: ${{ runner.os }}-${{ steps.diplomat-version.outputs.rev }}
+
       - name: Install Diplomat
+        if: steps.diplomat-cache.outputs.cache-hit != 'true'
```

Comment:
Question: did you test this to make sure that `!= 'true'` is the correct condition?  (I don't know if it's correct to quote the `true` since it is a boolean type in the docs)

Reply:
Yes, see https://github.com/actions/cache#example-workflow